src/audio/mixin_mixout/Kconfig

```diff
@@ -3,6 +3,7 @@
 config COMP_MIXIN_MIXOUT
 	tristate "Mixin_mixout component"
 	depends on IPC_MAJOR_4
+	default m if LIBRARY_DEFAULT_MODULAR
```

Comment:
(Regardless of loadable modules method) I see no reason in switching such frequently used module if we will load it always anyway.
I checked "reference firmware" and they are not using loadable option for typical modules, only for rarely used or external modules so I think we could optimize our settings similarly

Reply:
@abonislawski Today we are quite monolithic once FW and library has been loaded into SRAM, this works today, but is not fully optimal if we want to be more aggressive for PnP and offload more to DSP. Long term we need to optimize our hot code more to run on SRAM and cold code on IMR/DRAM, being able to unload (delete from SRAM and keep in IMR/DRAM) is needed if we want to maximize offload capability with limited SRAM. i.e. we can unload to swap media decoder modules (or other modules), power off SRAM banks when not used, change processing coefficients (e.g. switching SRC rates), unload init code (only used once), etc.